maintenance/wikia/addLayoutToPortableInfoboxesAllWikias.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Script goes through all portable infoboxes definitions on all wikias and adds stacked layout if one is not defined.
+ * Stacked layout was default so far and with next release tabular layout will be default
+ *
+ * @author Kamil Koterba
+ * @ingroup Maintenance
+ */
+
+require_once( dirname( __FILE__ ) . '/../Maintenance.php' );
+
+/**
+ * Maintenance script class
+ */
+class addLayoutToPortableInfoboxesAllWikias extends Maintenance {
+
+	public function __construct() {
+		parent::__construct();
+	}
+	
+	public function execute() {
+		$pages = $this->getPagesWithInfoboxes();
+		foreach ( $pages as $wikiaId => $pagesOnWikia ) {
+			$pagesParam = json_encode( $pagesOnWikia );
+			$cmd = 'SERVER_ID=' . $wikiaId . ' php maintenance/wikia/addLayoutToPortableInfoboxes.php --pages=' . $pagesParam;
+			$result = wfShellExec( $cmd );
+			$this->output( "$result\n" );
```

Comment:
Please consider adding debug info to the output here (A string describing what is being displayed in $result, i.e. "wfShellExec Output:" )